test/libsolidity/semanticTests/experimental/stdlib/cryptography.sol

```diff
@@ -0,0 +1,30 @@
+pragma experimental solidity;
+
+import "std/cryptography.sol";
+
+contract C {
+
+    function f(uint256 a) external returns (bytes32) {
+        return sha256(abi.encodePacked(a));
+    }
+
+    function g(uint256 a) external returns (bytes20) {
+        return ripemd160(abi.encodePacked(a));
+    }
+
+    function h(bytes32 h, uint8 v, bytes32 r, bytes32 s) public returns (address addr) {
+        return ecrecover(h, v, r, s);
+    }
```

Comment:
The test is here only to cover different import mechanisms, i.e. `import "std/cryptography.sol" vs `import { sha256 } from "std/cryptography.sol"`, I can remove it if it's superfluous. 

>By the way, is this little coverage really adequate for stdlib? 

No, these were just some sanity cases I added to the previous PR, since that one had only one test originally if I remember correctly. 

> I'd prefer if we at least had some mechanism to test that the new functions give the same outputs as the existing built-ins 

I agree, although I'm not really sure how to go about this, since if `pragma experimental solidity` is enabled, we shouldn't have access to the builtins (even though currently we do), which means we'd either have to have this as a unit test where we compile one source unit that uses builtins, and the other that uses stdlib, preserve the return values, and then compare them. `soltest`/`isoltest` doesn't allow this, as far as I know.